webpack.config.ts

```diff
@@ -36,11 +37,11 @@ function resolve(loggerName) {
             logger: path.resolve(__dirname, `src/logger/${loggerName}`)
         },
     }
-};
+}
 
 // ----- Configs ----- //
 
-const serverConfig = env => {
+const serverConfig = (env: { [key: string]: boolean | undefined }): Configuration => {
```

Comment:
Not proud of this one, but turns out `NodeJS.ProcessEnv` was the wrong one here after adding the `Configuration` return type. It seems to me that Webpack does the magic of parsing the CLI flags but I couldn't find a helpful type for it https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webpack/index.d.ts